playbooks/ansible/roles/samba.setup/tasks/cephfs/main.yml

```
@@ -7,7 +7,8 @@
       vars:
         prefix: "{{ role_path }}/tasks/cephfs/"
       with_first_found:
-        - files: "{{ [prefix] | product(config.os[config.nodes[inventory_hostname].os].includes) | map('join') | list }}"
+        - files: "{{ [prefix] | product(config.os[config.nodes[inventory_hostname].os].includes) | map('join')
+                  | list }}"
```

Since the overflow is very small, I think it will be cleaner to just move the full string to the next line:
```suggestion
        - files: >-
            {{ [prefix] | product(config.os[config.nodes[inventory_hostname].os].includes) | map('join') | list }}
```